src/crypto/hash/index.js

```diff
@@ -55,7 +56,9 @@ const hashFunctions = {
   sha256: nodeHash('sha256') || nobleHash(sha256, 'SHA-256'),
   sha384: nodeHash('sha384') || nobleHash(sha384, 'SHA-384'),
   sha512: nodeHash('sha512') || nobleHash(sha512, 'SHA-512'),
-  ripemd: nodeHash('ripemd160') || nobleHash(ripemd160)
+  ripemd: nodeHash('ripemd160') || nobleHash(ripemd160),
+  sha3_256: nobleHash(sha3_256),
+  sha3_512: nobleHash(sha3_512)
```

Comment:
SHA3 should also be natively implemented by NodeCrypto:
```suggestion
  sha3_256: nodeHash('sha3-256') || nobleHash(sha3_256),
  sha3_512: nodeHash('sha3-512') || nobleHash(sha3_512)
```

Reply:
Oops, so sorry about that! I should be more careful with force push. Thanks!!